### DISCUSSION TITLE
queryTxt ETIMEOUT problem

### DISCUSSION THREAD

**Discussion by altaygileren** [strapi/strapi#5962] [Legacy Topics]
Running into an issue with npm run develop where whenever it's restarting it gives me a queryTxt issue saying the connection is ETIMEOUT... I originally considered this may be an issue with MongoDB Atlas but it begins to work again after running npm run develop 2-3 times

**Comment by derrickmehaffy** [comment]
Bit of research this directly relates to Atlas:

https://stackoverflow.com/questions/56334603/cant-connect-to-mongodb-atlas
https://stackoverflow.com/questions/55499175/how-to-fix-error-querysrv-erefused-when-connecting-to-mongodb-atlas
https://stackoverflow.com/questions/38332841/error-connect-etimedout-54-162-26-9517185
https://stackoverflow.com/questions/49894913/error-at-connecting-to-mongodb-atlas-server

This error is simply being passed through mongoose and being logged to the console.
